classes/components/listPanels/PKPDoiListPanel.php

```diff
@@ -172,6 +177,12 @@ public function getConfig()
             'manager.dois.update.failedCreation',
             'plugins.importexport.common.action.export',
             'manager.dois.actions.description',
+            'publication.version',
```

Comment:
We need to remove this key from here, right?

Reply:
No, this is the locale key that needed to be transferred from OPS to pkp-lib. This is the `"Version 1"`, `"Version 2"`, etc string used in the version modal.